src/components/graph/menus/model-node-editor.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useState } from 'react';
+import { NativeSelect, TextField } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import { useIntl } from 'react-intl';
+import { updateTreeNode } from '../../../utils/rest-api';
+
+const ModelNodeEditor = ({ selectedNode }) => {
+    const intl = useIntl();
+
+    const [nameValue, setNameValue] = useState(selectedNode.data?.label);
+    const [modelType, setModelType] = useState(selectedNode.modelType);
+
+    const updateNameValue = (event) => {
+        setNameValue(event.target.value);
+    };
+
+    const updateModelType = (event) => {
+        setModelType(event.target.value);
+    };
+
+    const handleValidate = (event) => {
+        updateTreeNode({
+            id: selectedNode.id,
+            type: selectedNode.type,
+            name: nameValue,
+        }).then((response) => {
+            if (response.status !== 200) {
```

Comment:
snackbar

Reply:
Error message is now displayed in snackbar